homeassistant/components/workday/binary_sensor.py

```diff
@@ -72,16 +72,16 @@ async def async_setup_entry(
     province: str | None = entry.options.get(CONF_PROVINCE)
     sensor_name: str = entry.options[CONF_NAME]
     workdays: list[str] = entry.options[CONF_WORKDAYS]
+    language: str | None = entry.options.get(CONF_LANGUAGE)
```

Comment:
This is always set right? Why it is fetch as optional?

Reply:
Can also set it up without a country and then language is never configured hence the key is `None`